### ISSUE TITLE
Bug: disable sorting in `side-effect`

### ISSUE THREAD

**Issue by ai** [azat-io/eslint-plugin-perfectionist#48]
### Describe the bug

[`side-effect` imports](https://github.com/azat-io/eslint-plugin-perfectionist/issues/4) is unique. They change environment and can’t be sorted without potencial issue.

I suggets to disable sorting for them.

### Code example

import './enable-client.js'
import './add-feature-to-client.js'

### ESLint version

N/A

### ESLint Plugin Perfectionist version

v1.5.0

### Validations

- [X] Read the [docs](https://eslint-plugin-perfectionist.azat.io).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by ai** [comment]
`style` group also could have side-effects:

```js
import './reset.css'
import './index.css' // I is before R in alphabet
```

**Comment by azat-io** [comment]
Fixed in 01da88a779556d5955c8da9a7271995182890656

Released in [v1.5.1](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v1.5.1)
